coalib/output/ConsoleInteraction.py

```diff
@@ -278,15 +278,26 @@ def print_diffs_info(diffs, printer):
             color='green')
 
 
+_warn_deprecated_format_str = True  # Remove when format_str is deprecated
```

Comment:
Hm, but that wouldn't use our LogPrinter facility provided though the parameters... Are we willing to make that tradeoff?